.github/workflows/linux-ci.yml

```diff
@@ -57,19 +72,37 @@ jobs:
           cache: poetry
 
       - name: Install python deps
-        run: poetry install -E core -E backend
-
-      - name: Configure libfuse path
-        run: |
-          LIBFUSE_PATH=$(find /usr/lib -name '*fuse*.so*' 2>/dev/null | head -n 1)
-          echo "::warning title=Libfuse path::use lib found at $LIBFUSE_PATH"
-          echo "FUSE_LIBRARY_PATH=$LIBFUSE_PATH" >> "$GITHUB_ENV"
+        run: poetry install -E core -E backend --remove-untracked
 
       - name: Generate PyQT GUI data
         run: poetry run python misc/generate_pyqt.py
 
       - name: Install libparsec_python
+        if: matrix.binding == 'rust-binding'
         run: poetry run pip install oxidation/libparsec_python
```

Comment:
`pip install` will generate a release build, we may want to use a debug build which is faster to build

Reply:
but then it should be faster to test right?